run/grpc-hello-world-streaming/README.md

```diff
@@ -0,0 +1,28 @@
+gRPC Kotlin Streaming Cloud Run Example
```

Comment:
Should we make this a BiDi stream? Either that or specify it's a "server streaming" example?

Reply:
Updating the description to "server streaming" because Cloud Run doesn't yet support bidi streams.